packages/web-components/src/components/va-banner/test/va-banner.e2e.ts

```diff
@@ -0,0 +1,123 @@
+import { newE2EPage } from '@stencil/core/testing';
+import { axeCheck } from '../../../testing/test-helpers';
+
+describe('va-banner', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent('<va-banner></va-banner>');
+    const element = await page.find('va-banner');
+
+    expect(element).toEqualHtml(`
+     <va-banner class="hydrated" data-e2e-id="emergency-banner">
+       <mock:shadow-root>
+         <va-alert class="hydrated" full-width="" status="info">
+           <h3 slot="headline"></h3>
+           <slot></slot>
+         </va-alert>
+        </mock:shadow-root>
+      </va-banner>
+    `);
+  });
+
+  it('renders an empty shadow root when not visible', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent('<va-banner visible="false"></va-banner>');
+    const element = await page.find('va-banner');
+
+    expect(element).toEqualHtml(`
+      <va-banner class="hydrated" visible="false">
+        <mock:shadow-root></mock:shadow-root>
+      </va-banner>
+    `);
+  });
+
+  it('renders an shadow root div when not visible', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent('<va-banner visible="false"></va-banner>');
+    const element = await page.find('va-banner');
+
+    expect(element).toEqualHtml(`
+      <va-banner class="hydrated" visible="false">
+        <mock:shadow-root></mock:shadow-root>
+      </va-banner>
+    `);
+  });
```

Comment:
Are these supposed to be checking different things?

Reply:
Accidental duplicate. I removed the second test